### PR TITLE
Notices page for IPS

### PR DESCRIPTION
**Pull request by vedangimittal**
- Updated Notices page for IPS
- Defect-https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=627532
- Jira story- https://jsw.ibm.com/browse/PFEBMC-2686

### CODE REVIEW

**Comment by vedangimittal** [comment]
Screenshot of the change

Screenshot 2024-06-24 at 8 45 52 PM

File: src/views/Notices/Notices.vue [hunk 1]
```diff
@@ -21,5 +21,6 @@ export default {
 .notice {
   overflow-x: hidden;
   white-space: break-spaces;
+  text-align: justify;
```
Comment (Nikhil-Ashoka) [inline src/views/Notices/Notices.vue:24]: Why is this change required?

Comment (vedangimittal) [reply src/views/Notices/Notices.vue:24]: Looks more clean

Before
Screenshot 2024-06-25 at 1 14 12 PM


After
Screenshot 2024-06-25 at 1 13 25 PM

Comment (Nikhil-Ashoka) [reply src/views/Notices/Notices.vue:24]: Okay

**Review by Renuka9527 — APPROVED** [review]

**Review by nabilpandiyalayil — APPROVED** [review]

**Review by Nikhil-Ashoka — APPROVED** [review]
